PhotoPlusFrontend/PhotoPlus/src/app/components/admin-panel/subcomponents/change-privileges/change-privileges.component.ts

```diff
@@ -24,7 +24,7 @@ export class ChangePrivilegesComponent implements OnInit {
     this.filteredUsers = new Array<BehaviorSubject<User>>();
     this.userService.getAll().subscribe((data) => {
       for (let user of data) {
-        if(user.name === "admin"){
+        if(user.login === "admin"){
```

Comment:
maybe (user.role=== "admin")?

Reply:
No i wanted to disallow changing the role of builtin user.